test/tokenService.js

```diff
@@ -422,11 +677,29 @@ describe('Interchain Token Service', () => {
                 .and.to.emit(gateway, 'ContractCall')
                 .withArgs(service.address, chain, service.address.toLowerCase(), keccak256(payload), payload);
         });
+
+        it('Should revert on remote custom token manager deployment if paused', async () => {
```

Comment:
this should be a separate describe test so you don't have to set it to unpause for every other test

Reply:
will maybe fix this later, noted